process/smartContract/processorV2/processV2.go

```diff
@@ -1474,7 +1474,8 @@ func (sc *scProcessor) processIfErrorWithAddedLogs(acntSnd state.UserAccountHand
 
 	userErrorLog := createNewLogFromSCRIfError(scrIfError)
 
-	if !sc.isInformativeTxHandler(scrIfError) {
+	isRecvSelfShard := sc.shardCoordinator.SelfId() == sc.shardCoordinator.ComputeId(scrIfError.RcvAddr)
```

Comment:
is this condition sufficient? I remember the case was invalid with the receiver in another shard.

Reply:
I think the condition below will be easier to read if we check with a isReceiverOtherShard instead of !isReceiverSelfShard